examples/pennylane/6_Adjoint_gradient_computation/6_Adjoint_gradient_computation.ipynb

```diff
@@ -22,7 +22,7 @@
    "cell_type": "markdown",
```

Comment:
<p>The example notebook link that you have been removed should be replaced with <code>pennylane/3_Hydrogen_Molecule_geometry_with_VQE/3_Hydrogen_Molecule_geometry_with_VQE.ipynb</code>. File was renamed with this PR: <a href="https://github.com/amazon-braket/amazon-braket-examples/pull/194/files" target="_blank">https://github.com/amazon-braket/amazon-braket-examples/pull/194/files</a>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/536/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Good catch! Addressed.